src/runtime_src/core/common/config_reader.h

```diff
@@ -156,7 +156,7 @@ get_data_transfer_trace()
 inline bool
 get_power_profile()
 {
-  static bool value = get_profile() && detail::get_bool_value("Debug.power_profile",false);
+  static bool value = detail::get_bool_value("Debug.power_profile",false);
```

Comment:
If we're not going to use get_profile() as a top-level gating mechanism, then we should only use it for host profiling, change the name to get_host_profile(), and only expose individual types of profiling separately. Perhaps not something for this checkin, but a down the road issue to address.

Reply:
@pgschuey this previously also was used for host profiling as you needed other switches to get device data_transfers etc. It might be a bad idea to break backwards compatibility with last 5 releases.